src/utility/Albany_ThyraCrsMatrixFactory.hpp

```diff
@@ -43,7 +47,7 @@ struct ThyraCrsMatrixFactory {
 
   bool is_filled () const { return m_filled; }
 
-  bool is_static_profile () const { return m_static_profile; } 
+  bool is_static_profile () const { return true; }
```

Comment:
Are we using static profile always, even for Epetra? If so, then no point in keeping this method, we should purge it.

Reply:
OK, I'll purge it. Hope it's not used anywhere.